### PR TITLE
No need to involve the manager. Fabienne will manage the paper work f…

### PR DESCRIPTION
**Pull request by sylvestre**
…or us

### CODE REVIEW

File: src/data/results.html [hunk 1]
```diff
@@ -74,12 +74,10 @@ <h2 lang='en'>Monitoring form of working hours capped in days over the year</h2>
         </dl>
       </section>
       <p>
-        A retourner à Doris Coleman chaque mois, après avoir été validé par votre
-        supérieur hiérarchique.
+        A retourner à Fabienne chaque mois. Le <i>country ambassador</i> signera le document par la suite.
```
Comment (n1k0) [inline src/data/results.html]: Nit: À

**Comment by magopian** [comment]
Should we have the email addresses here? (commenting after the battle :D

**Comment by Natim** [comment]
It could be nice to have a french-monthly-pto@mozilla.com alias

**Comment by Natim** [comment]
Oh I didn't see the update was on master...

**Pull request merged**